src/Xamarin.Android.Tools.Bytecode/ClassPath.cs

```diff
@@ -233,38 +233,9 @@ void FixupParametersFromDocs (XElement api)
 			}
 		}
 
-		JavaDocletType GetDocletType (string path)
-		{
-			var kind = JavaDocletType.DroidDoc;
-			char [] buf = new char [500];
-			string packagesHtml = Path.Combine (path, "packages.html");
-			if (File.Exists (packagesHtml) && File.ReadAllText (packagesHtml).Contains ("<body class=\"gc-documentation develop reference api "))
-				kind = JavaDocletType.DroidDoc2;
-			using (var reader = File.OpenText (Path.Combine (path, "index.html")))
-				reader.ReadBlock (buf, 0, buf.Length);
-			string rawHTML = new string (buf);
-			if (rawHTML.Contains ("Generated by javadoc (build 1.6"))
-				kind = JavaDocletType.Java6;
-			else if (rawHTML.Contains ("Generated by javadoc (version 1.7"))
-				kind = JavaDocletType.Java7;
-			else if (rawHTML.Contains ("Generated by javadoc (1.8"))
-				kind = JavaDocletType.Java8;
-
-			// Check to see if it's an api.xml formatted doc
-			string rawXML = null;
-			using (var reader = File.OpenText (path)) {
-				int len = reader.ReadBlock (buf, 0, buf.Length);
-				rawXML = new string (buf, 0, len);
-			}
-			if (rawXML.Contains ("<api>") && rawXML.Contains ("<package"))
-				kind = JavaDocletType._ApiXml;
-
-			return kind;
-		}
-
 		IAndroidDocScraper CreateDocScraper (string src)
 		{
-			switch (DocletType ?? GetDocletType (src)) {
+			switch (DocletType ?? AndroidDocScraper.GetDocletType (src)) {
```

Comment:
It's good that we updated `class-parse` so that it doesn't *always* set `DocletType`, but...do we really want to prefer it?

I'm not convinced that we do.